qa/tasks/userdata_setup.yaml

```diff
@@ -1,25 +1,32 @@
-#cloud-config-archive
+#cloud-config
 
-- type: text/cloud-config
-  content: |
-    output:
-      all: '| tee -a /var/log/cloud-init-output.log'
+packages:
+  - xfslibs-dev
```

Comment:
Way to specific -- the QEMU task shouldn't be designed to handle only xfstest cases